packages/components/src/Card/_docs/Card.mdx

```diff
@@ -22,6 +22,9 @@ The `Card` component is a flexible container used to wrap primary content. It ha
 
 ## API
 
-<DocsStory of={CardStories.Variants} />
+### Card
+<Canvas of={CardStories.Colors} />
```

Comment:
I think we need to update the story name since the docs are currently throwing undefined on this since the title is `color` :)